hack/deploy-setup.sh

```diff
@@ -17,7 +17,7 @@ load_manifest() {
 
   pushd ${repo}/deploy
     if ! ${OPERATOR_EXEC} get ns $2 > /dev/null 2>&1 && test -f namespace.yaml ; then
-      
```

Comment:
Please clean up this file.

Reply:
Let's not touch this file in this commit. Push a separate cleanup PR if you want to clean this up.